### PR TITLE
adding game files and .gitignore, updating README

### PR DESCRIPTION
**Pull request by ShashurynVitalii**
The game was developed with use of pygame framework. It definitely can run on micro computers like Raspberry Pi, but I am not sure about other hardware. If it is considered as a badly or wrongly completed task, then you can asses it as 0 point. But I still want to do at least something in order to have some points or at least get access to exam

Also in case if this task will be considered as wrongly done then will I still be able to commit next tasks?

Sorry, if I am asking stupid questions that already were asked. Also I am not planning to defend my tasks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-∙ distance;
-
-∙ tool size;
-
-∙ win statement;
-
-∙ lose statement;
-
-∙ AI parameters
+## General info about project
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:2]: which code is for HW?

File: README.md [hunk 2]
```diff
@@ -28,14 +28,15 @@ your turn click left mouse button one more time to let computer oponent perform
 ## Instalation and run
 
 In order to run the game you firstly will need to install [Python](https://www.python.org/downloads/)\
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:30]: what task is it?

Comment (ShashurynVitalii) [reply README.md:30]: task 2

File: README.md (hunk 1 above)
Comment (andrii-s-shpitser-lpnu-ua) [reply README.md:2]: which code is for HW?

Comment (ShashurynVitalii) [reply README.md:2]: second commit, server folder (serial_connection.py, server.ino files)

Comment (andrii-s-shpitser-lpnu-ua) [reply README.md:2]: how you can run py on HW? which type of HW are you using?

Comment (ShashurynVitalii) [reply README.md:2]: I made a mistake placing both files in one folder, .py file is for client side

Comment (andrii-s-shpitser-lpnu-ua) [reply README.md:2]: so: what code is for HW?

Comment (ShashurynVitalii) [reply README.md:2]: server.ino file for arduino

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**